Update ready for release `v1.1.0` (#224)
Azure/terraform-azurerm-caf-enterprise-scale: README.md

## Release Notes

Release `v1.1.0` introduces the following changes:

> - **BREAKING CHANGE**: Replace `Deploy-ASC-Configuration` Policy Assignment with `Deploy-ASCDF-Config`, utilizing built-in policies and adding support for [Microsoft Defender for open-source relational databases](https://docs.microsoft.com/azure/defender-for-cloud/defender-for-databases-introduction).

Reviewer: ```suggestion
- **BREAKING CHANGE**: Replaced `Deploy-ASC-Configuration` Policy Assignment with `Deploy-ASCDF-Config`, utilizing built-in policies and also adds support for [Microsoft Defender for open-source relational databases](https://docs.microsoft.com/azure/defender-for-cloud/defender-for-databases-introduction).
```

>   - Fixed [Add Defender support for Open-source relational databases #131](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/issues/131).

Reviewer: ```suggestion
  - Fixing [Add Defender support for Open-source relational databases #131](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/issues/131).
```

>   - Will result in loss of policy compliance history.

Reviewer: ```suggestion
  - **Note:** Will result in loss of policy compliance history.
```
